exchanges/bittrex/bittrex_types.go

```diff
@@ -209,3 +209,18 @@ type WithdrawalHistory struct {
 		InvalidAddress bool    `json:"InvalidAddress"`
 	} `json:"result"`
 }
+
+// DepositHistory holds the Deposit history data
+type DepositHistory struct {
+	Success bool   `json:"success"`
+	Message string `json:"message"`
+	Result  []struct {
+		Id            int64   `json:"Id"`
```

Comment:
change id -> ID

Reply:
I overlooked that in the previous PR commit (4b9de0fdd02ca55d4e97df8337b2439bf72805ae)  "Id"  was a changed ti "ID". So I can make changes in next PR or author can make correction on current PR?

Reply:
Awesome, just needs a rebase or merge and we are good to go. Thanks heaps for the PR!

Reply:
And you know ... I’m now studying the Bitrix API documentation, and I have the feeling that their engineer is well versed in cannabis :)

Reply:
@vazha seems as though a lot of the exchange API documentation authors are high flyers 😆